ISSUE_POLICY.md

```diff
@@ -0,0 +1,7 @@
+Before filing an issue, make sure to search for related issues and check if
+they address yours. Our policy is that GitHub issues are
+for bug reports, documentation fixes, feature requests, 
+and installation issues. Please do not delete the issue template placeholder unless you are certain your
+issue is outside its scope.
+  
+For support (ex. "How do I do X?"), please refer to [Stack Overflow](https://stackoverflow.com/questions/tagged/mlflow).
```

Comment:
Maybe move this up to the top (i.e. right after the "Before filing an issue" sentence) so people are more likely to see it.